Closes #114 - Add gages to Fire Hydrology - Query by Fire Perimeter workflow

amedenblik opened the pull request:
Closes #114

harper-wavra commented on src/app/map/map.component.ts:
```diff
@@ -625,6 +625,7 @@ export class MapComponent implements OnInit {
     } 
     this.count ++;
     this.checkCount(this.count, 6);
+    this._mapService.map.fitBounds(this.firePerimeterLayer.getBounds(), {padding: [75,75]});
```
There seem to be console errors thrown at this line, maybe we can check if there is a `firePerimeterLayer` before fitting bounds. 
![image](https://user-images.githubusercontent.com/59063181/208715118-d16ae058-a96b-4b0f-be77-a6a415c4643b.png)

harper-wavra reviewed (changes requested):
I know it hasn't passed linting yet but thought I would review now as I will be out for a bit after today.

Working great! Just a couple comments, 

-The gage table seems to be a bit off. 
![image](https://user-images.githubusercontent.com/59063181/208715850-57b2b738-c7f5-4ba0-84fe-8a3538d00edf.png)
- a couple console.logs 

I also wonder if right before we merge this, we update the polygon endpoints & change the `nldiPolygonQuery` url back so everything is done in this one issue

amedenblik commented:
I have addressed linting and your review. I also reverted `nldiPolygonQuery` back to `https://streamstats.usgs.gov/nldipolygonservices/nldi_poly_query`. Let's ping @Anders-Hopkins to rename `https://streamstats.usgs.gov/nldipolygonservices/nldi_poly_query_new` to `https://streamstats.usgs.gov/nldipolygonservices/nldi_poly_query` before we merge this PR.

amedenblik commented:
@harper-wavra this branch is working again and ready to be reviewed

harper-wavra reviewed (changes requested):
I tested each endpoint. Delineation and Query by fire perimeter both looked & worked well. Query by basin seems to be getting hung up though. I tried on the https://test.streamstats.usgs.gov/national/ site and it's working there, so I think it might be related to changes in this branch.

amedenblik commented:
I pulled in the `dev` branch, which fixed the Query by Basin issues. Reminder- let's not merge until nldi_poly_query is renamed.

harper-wavra reviewed (approved):
Looks good!
